tsconfig.json

```diff
@@ -0,0 +1,57 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
```

Comment:
You can just remove the commented out stuff

Reply:
lol.. i kinda like it since it lets me, as a total TS newb, no what's possible.